src/cli/timing_tests.cpp

```diff
@@ -57,9 +57,9 @@ class Timing_Test
       virtual ~Timing_Test() {}
 
       std::vector<std::vector<ticks>>
-      execute_evaluation(const std::vector<std::string>& inputs,
-                         size_t warmup_runs,
-                         size_t measurement_runs);
+                                   execute_evaluation(const std::vector<std::string>& inputs,
```

Comment:
Most of these not good looking things are result of "inherited" formating. It should be easy to fix. I will care about it.